.github/workflows/visual-regression.yml

```
@@ -0,0 +1,99 @@
+          enable-commit-comment: false
+          overwrites-pull-request-comment: true
+        env:
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_VISUAL_REGRESSION_SITE_ID }}
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+        timeout-minutes: 1
+      - name: Generate the visual regression message.
+        id: visual-regression-message
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
```

actionlint (1.7.4)

property "netlify-deploy-message" is not defined in object type {netlify-visual-regression: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)